### PR TITLE
Add test for the records check page.

### PR DESCRIPTION
**Pull request by MancunianSam**
There are two tests. One to check to see if the progress bars are there
and one to check that the redirect happens when the checks are complete.

The progress bars check will need to be updated when the file format
progress bar is added.

I've also added a line to the full end to end test to check for the file check results page.

### CODE REVIEW

File: src/test/resources/features/Fullflow.feature [hunk 1]
```diff
@@ -19,3 +19,4 @@ Feature: Full user journey
     When the user selects directory containing: testfile1
     And the user clicks the continue button
     Then the user will be on a page with the title Checking your records
+    And the user will be on a page with the title Record check results
```
Comment (TomJKing) [inline src/test/resources/features/Fullflow.feature:22]: Missing line break.

File: src/test/resources/features/RecordsCheck.feature [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Feature: Records Check Page
+
+  Scenario: User sees the file check progress bars
+    Given A logged out user
+    And an existing consignment for transferring body MOCK1
+    And an existing transfer agreement
+    And the user is logged in on the upload page
+    When the user selects directory containing: testfile1
+    And the user clicks the continue button
+    Then the user should see the AV Metadata progress bar
```
Comment (TomJKing) [inline src/test/resources/features/RecordsCheck.feature:10]: Maybe put "AV" out in full, makes the test clearer: `Then the user should see the Antivirus Metadata progress bar `

File: src/test/resources/features/RecordsCheck.feature [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    And the user should see the checksum progress bar
+
+  Scenario: User is redirected to results page when the record checks are complete
+    Given A logged out user
+    And an existing consignment for transferring body MOCK1
+    And an existing transfer agreement
+    And the user is logged in on the upload page
+    When the user selects directory containing: testfile1
+    And the user clicks the continue button
+    Then the user will be on a page with the title Record check results
```
Comment (TomJKing) [inline src/test/resources/features/RecordsCheck.feature:20]: Missing line break

**Review by TomJKing — COMMENTED** [review]
Looks good. Couple of minor comments/suggestions

File: src/test/resources/features/RecordsCheck.feature (hunk 2 above)
Comment (MancunianSam) [reply src/test/resources/features/RecordsCheck.feature]: I've changed it to Antivirus Metadata.
It does mean I had to change step to use the match expression rather than `progressBarName.replaceAll("\\s", "-").toLowerCase` but I think this is clearer now. It means we can add other progress bar elements without worrying about what the id in the front end is as well.

File: src/test/resources/features/RecordsCheck.feature (hunk 3 above)
Comment (MancunianSam) [reply src/test/resources/features/RecordsCheck.feature]: You'd be disappointed if I didn't leave at least one line break out :smile:

File: src/test/resources/features/Fullflow.feature (hunk 1 above)
Comment (MancunianSam) [reply src/test/resources/features/Fullflow.feature]: I've added a line break in here.

**Review by TomJKing — APPROVED** [review]

File: src/test/resources/features/Fullflow.feature [hunk 4]
```diff
@@ -18,6 +18,8 @@ Feature: Full user journey
     Then the user will be on a page with the title "Upload Records"
     When the user selects directory containing: testfile1
     And the user clicks the continue button
-    Then the user will be on a page with the title "Checking your records"
+    Then the user will be on a page with the title Checking your records
     And the av metadata progress bar should be visible
```
Comment (TomJKing) [inline src/test/resources/features/Fullflow.feature:22]: Maybe reword so "av" is out in full: `And the Antvirus metadata progress bar should be visible` and reuse the logic from the new step you introduced: `Then the user should see the Antivirus Metadata progress bar`

**Review by TomJKing — COMMENTED** [review]
One minor comment

File: src/test/resources/features/Fullflow.feature (hunk 4 above)
Comment (MancunianSam) [reply src/test/resources/features/Fullflow.feature]: Good spot, cheers. I've changed it over.

**Review by TomJKing — APPROVED** [review]
